### ISSUE TITLE
Missing vSphere Data Source: Datastore

### ISSUE THREAD

**Issue by austin-millan** [starhawking/python-terrascript#41]
Terrascript is missing the [vsphere_datastore](https://www.terraform.io/docs/providers/vsphere/d/datastore.html) data source.

This declaration should be included in: `python-terrascript/terrascript/vsphere/d.py`

```
class vsphere_datastore(_data): pass
datastore = vsphere_datastore
```

**Comment by mjuenema** [comment]
I moved house last week and my computers are still in boxes. So please be a bit patient before I can address this issue. Thanks, Markus.

**Comment by austin-millan** [comment]
Issue addressed in #52.
